### PR TITLE
Translation of Caching chapter to French 

### PR DESCRIPTION
**Pull request by allemas**
New pull request with reviews and page generation :) 

Makes progress on #539

### CODE REVIEW

File: src/content/fr/2019/caching.md [hunk 1]
```diff
@@ -0,0 +1,691 @@
...
+Pour qu'un client HTTP mette en cache une ressource, il doit répondre a deux questions&nbsp;:
+
+*   "Combien de temps dois-je mettre en cache&nbsp;?"
+*   "Comment puis-je valider que le contenu est encore frais&nbsp;?"
+
+Lorsqu'un navigateur Web envoie une réponse à un client, il inclut généralement dans sa réponse des en-têtes qui indiquent si la ressource peut être mise en cache, pour combien de temps et quel est son âge. La RFC 7234 traite plus en détail de ce point dans la section [4.2 (Freshness)](https://tools.ietf.org/html/rfc7234#section-4.2) et [4.3 (Validation)](https://tools.ietf.org/html/rfc7234#section-4.3).
+
+Les en-têtes de réponse HTTP généralement utilisées pour transmettre la durée de vie sont&nbsp;:
+
+*   <i lang="en">`Cache-Control`<span> vous permet de configurer la durée de vie du cache (c'est-à-dire sa durée de validité).
```
Comment (bazzadp) [inline src/content/fr/2019/caching.md]: Typo here (think it's my fault btw!)

```suggestion
*   <i lang="en">`Cache-Control`</i> vous permet de configurer la durée de vie du cache (c'est-à-dire sa durée de validité).
```

File: src/content/fr/2019/caching.md [hunk 2]
```diff
@@ -0,0 +1,691 @@
...
+  </a>
+  <div id="fig3-description" class="visually-hidden">Un graphique à barres superposées montrant que 20&nbsp;% des réponses pour ordinateurs de bureau ne peuvent être mises en cache, 47&nbsp;% ont un cache supérieur à zéro, 27&nbsp;% sont mises en cache de manière heuristique et 6&nbsp;% ont un TTL de 0. Les statistiques pour les mobiles sont très similaires (19&nbsp;%, 47&nbsp;%, 27&nbsp;% et 7&nbsp;%)</div>
+  <figcaption id="fig3-caption">Figure 3. Distribution des réponses pouvant être mises en cache.</figcaption>
+</figure>
+
+Le tableau ci-dessous détaille les [TTL](https://fr.wikipedia.org/wiki/Time_to_Live) du cache pour les requêtes en provenance d'ordinateurs de bureau. La plupart des types de contenu sont mis en cache, mais les ressources CSS semblent toujours être mises en cache à des valeurs TTL élevées.
```
Comment (bazzadp) [inline src/content/fr/2019/caching.md]: Doesn't look like my previous comment was addressed here:

I see you've included a wikipedia reference here thats not in the English version. If you think things like this are valuable then I would encourage you to also include such changes to the English version i) to keep them in synch and ii) to benefit English speakers. Similarly if you spot typos while translating.

However in this specific case I would question its value. The [French version](https://fr.wikipedia.org/wiki/Time_to_Live) talks about IP TTL and DNS TTL, but not HTTP TTL which we are discussing here. Funnily enough, [the English version of that page](https://en.wikipedia.org/wiki/Time_to_live#HTTP) _does_ talk about HTTP TTL but even then only covers it very briefly so not sure if that link is more confusing than beneficial?

To be honest I'd remove this. However if you really want it, then a better link might be the MDN docs (in [French](https://developer.mozilla.org/fr/docs/Glossaire/TTL#Mise_en_cache) and [English](https://developer.mozilla.org/en-US/docs/Glossary/TTL#Caching))?

```suggestion
Le tableau ci-dessous détaille les TTL du cache pour les requêtes en provenance d'ordinateurs de bureau. La plupart des types de contenu sont mis en cache, mais les ressources CSS semblent toujours être mises en cache à des valeurs TTL élevées.
```

**Comment by allemas** [comment]
I didn't find the last change requested :(

**Comment by bazzadp** [comment]
Yeah was all hidden away in the review: https://github.com/HTTPArchive/almanac.httparchive.org/pull/669#discussion_r382925557

LGTM so let's merge!

**Pull request merged**